src/data/faq_de.json

```diff
@@ -843,7 +843,7 @@
                         "active": false,
                         "textblock": [
                             "In Fall eines positiven Testergebnisses sollten Sie Ihr Testergebnis teilen und umgehend einen PCR-Test machen, um das Ergebnis zu bestätigen oder zu widerlegen. Bei einem positiven PCR-Test sollten Sie das Testergebnis erneut teilen, um Kontakte zu warnen, die Sie möglicherweise zwischen den beiden Tests getroffen haben. Voraussetzung dabei ist, dass die Kontaktpersonen ebenfalls die Corona-Warn-App nutzen und die Risikoberechnung ein hohes Risiko bestimmt, basierend auf Dauer und Abstand des Kontakts. Sollte der PCR-Test negativ sein, haben Sie nicht die Möglichkeit, Ihren positiven Schnelltest und die Warnung zurückzuziehen.",
-                            "<a href='../../assets/documents/DE_CWA_RKI-Plakat_Schnelltests_v2-kl.pdf' target='_blank'><img width= 300px src='../../assets/documents/DE_CWA_RKI-Plakat_Schnelltests_v2-kl-1.PNG' alt='PDF'></a>",
+                            "<a href='../../assets/documents/DE_CWA_RKI-Plakat_Schnelltests_v2-kl.pdf' target='_blank'><img width= 300px src='../../assets/documents/DE_CWA_RKI-Plakat_Schnelltests_v2-kl-1.PNG' alt=''></a>",

```

Comment:
Not sure about this one. This seems to be an empty link because the link has no text, only an image. It would make sense to add something useful here.